trcli/constants.py

```
@@ -86,12 +94,13 @@
 )
 
 TOOL_VERSION = f"""TestRail CLI v{trcli.__version__}
-Copyright 2024 Gurock Software GmbH - www.gurock.com"""
+Copyright 2025 Gurock Software GmbH - www.gurock.com"""
 TOOL_USAGE = f"""Supported and loaded modules:
     - parse_junit: JUnit XML Files (& Similar)
     - parse_robot: Robot Framework XML Files
     - parse_openapi: OpenAPI YML Files
-    - add_run: Create a new test run"""
+    - add_run: Create a new test run
+    - labels: Manage labels (projects, cases, and tests)"""
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)